src/components/modules/Leaderboard/TopKulaCard.tsx

```diff
@@ -0,0 +1,33 @@
+import { IStackingUser } from '../../../types/interfaces'
+import { toShortAddress } from '../../../utils/account'
+import { currency } from '../../../utils/Number'
+
+interface TopKulaCardProps {
```

Comment:
We should not use Kula or xKula in the source code. Because we can rebrand or rename the project in the furture. Should we use Token, xToken, or something general?

Reply:
i fixed